src/actions/slack/legacy_slack/slack.ts

```diff
@@ -28,9 +28,13 @@ https://github.com/looker/actions/blob/master/src/actions/slack/legacy_slack/REA
 
   async form(request: Hub.ActionRequest) {
     const form = new Hub.ActionForm()
+    let channelType = "channels"
+    if (request.formParams.channelType !== undefined && request.formParams.channelType === "users") {
```

Comment:
nit: `request.formParams.channelType !== undefined` can be `isNil` in lodash, and that'd handle `null` and `undefined` too.